### PR TITLE
Fixed readme on client changes.

### PR DESCRIPTION
**Pull request by sima-zhu**

### CODE REVIEW

File: sdk/search/azure-search-documents/README.md [hunk 1]
```diff
@@ -32,59 +32,86 @@ create and manage indexes, load data, implement search features, execute queries
 In order to interact with the Azure Cognitive Search service you'll need to create an instance of the Search Client class. 
 To make this possible you will need an [api-key of the Azure Cognitive Search service](https://docs.microsoft.com/en-us/azure/search/search-security-api-keys).
 
-The SDK provides two clients.
+The SDK provides three clients.
 
-1. SearchIndexClient for all document operations.
-2. SearchServiceClient for all CRUD operations on service resources.
+1. SearchIndexClient for all CRUD operations on index and synonym maps.
+1. SearchIndexClient for all CRUD operations on indexer, date source, and skillset.
```
Comment (alzimmermsft) [inline sdk/search/azure-search-documents/README.md:38]: Repeated `SearchIndexClient`, should this be `SearchIndexerClient`?

File: sdk/search/azure-search-documents/README.md [hunk 2]
```diff
@@ -32,59 +32,86 @@ create and manage indexes, load data, implement search features, execute queries
...
+1. SearchClient for all document operations.
 
-#### Create a SearchServiceClient
+#### Create a SearchIndexerClient
```
Comment (alzimmermsft) [inline sdk/search/azure-search-documents/README.md:41]: This section is showing the creation of a `SearchIndexClient`.

```suggestion
#### Create a SearchIndexClient
```

File: sdk/search/azure-search-documents/README.md [hunk 3]
```diff
@@ -32,59 +32,86 @@ create and manage indexes, load data, implement search features, execute queries
...
 
-Once you have the values of the Azure Cognitive Search service [URL endpoint](https://docs.microsoft.com/en-us/azure/search/search-create-service-portal#get-a-key-and-url-endpoint) 
-and [admin key](https://docs.microsoft.com/en-us/azure/search/search-security-api-keys) you can create the SearchServiceClient:
+To create a SearchIndexClient, you will need an existing index name as well as the values of the Azure Cognitive Search service 
+[URL endpoint](https://docs.microsoft.com/en-us/azure/search/search-create-service-portal#get-a-key-and-url-endpoint) and 
+[query key](https://docs.microsoft.com/en-us/azure/search/search-security-api-keys).
+Note that you will need an admin key to index management (query keys only work for queries).
```
Comment (alzimmermsft) [inline sdk/search/azure-search-documents/README.md:46]: Should this bit be added above showing creation of clients as it is common for all of them?

File: sdk/search/azure-search-documents/README.md [hunk 4]
```diff
@@ -32,59 +32,86 @@ create and manage indexes, load data, implement search features, execute queries
...
+    .credential(new AzureKeyCredential(apiKey))
+    .buildAsyncClient();
+```
 
-public void createAsyncIndexClient() {
-    SearchAsyncClient searchAsyncClient = new SearchClientBuilder()
-        .endpoint(endpoint)
+#### Create a SearchClient
+
+Once you have the values of the Azure Cognitive Search service [URL endpoint](https://docs.microsoft.com/en-us/azure/search/search-create-service-portal#get-a-key-and-url-endpoint)
```
Comment (alzimmermsft) [inline sdk/search/azure-search-documents/README.md:93]: Should mention for `SearchClient` that you need the name of an index that has already been created.

File: sdk/search/azure-search-documents/README.md (hunk 3 above)
Comment (sima-zhu) [reply sdk/search/azure-search-documents/README.md]: Done.

**Pull request merged**